README.md

```diff
@@ -729,6 +729,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Numan Javed](https://github.com/numanjvd82)
   - [Nur Akmal Mohd Nordi](https://github.com/nurakmal920207)
   - [Nworie Chikwado Emmanuel](https://github.com/chikwado17)
+  - [Nitin Sharma](https://github.com/nitin-787)
```

Comment:
Please add your name in line 727, after Nithin.
Let us know if you need any help with that! 🙂


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix